app/Http/Controllers/Backend/DashboardController.php

```diff
@@ -5,16 +5,85 @@
 use Illuminate\Http\Request;
 use App\Http\Requests;
 use App\Http\Controllers\Controller;
+use App\Models\Backend\Order;
+use DB;
 
 class DashboardController extends Controller
 {
     /**
      * Index admmin
      *
+     * @param string $condition this is of group by
+     *
+     * @return view index of admin
+     */
+    public function index($condition = 'DAYNAME')
+    {
+        // create array save data report
+        $data = array();
+
+        // check condition group by.
+        // only group by with condition = DAYNAME or MONTHNAME of created_at
+        // if condition != DAYNAME or MONTHNAME then group by with DAYNAME
+        if ($condition == 'DAYNAME' or $condition == 'MONTHNAME') {
+            $data['order']   = $this->getReportOrder($condition);
+            $data['user']    = $this->getReportUser($condition);
+            $data['product'] = $this->getReportProduct($condition);
+        } else {
+            $data['order']   = $this->getReportOrder('DAYNAME');
+            $data['user']    = $this->getReportUser('DAYNAME');
+            $data['product'] = $this->getReportProduct('DAYNAME');
+        }
+        
+        return view('backend.dashboard.index', compact('data'));
```

Comment:
[rank A] you should use:
`return view('backend.dashboard.index')->with($data);`
and on the view, we can get parameters as: order[s], user[s], product[s] (use plural for array/list variables)